Sort Day Overview by time of day and postcode

mbarton opened the pull request:
Ronseal pretty much

adamcunnington commented:
Hey @mbarton feel free to review and tweak test data values as you like.

It's [here](https://docs.google.com/spreadsheets/d/1eMiA6DAYmYJVYwNvxj18yRiSBImCXu6WiCD-4CqM7P8/edit#gid=862565527)

mbarton commented:
Perfect thanks! I've updated all the existing data to have fake UK postcodes.

I realised my solution wasn't working because I'd modified the driver overview code 🤦. I've fixed it to modify the day overview instead as requested.

@adamcunnington I remember you mentioning there was a decorator somewhere that could help? I couldn't remember exactly so I went with using a Pandas df to modify the data before passing to render, as we do above in the driver overview

adamcunnington commented:
This is great. The decorator won't work because the return type is different so the way you've done it is fine.
